docs/training-debug/index.txt

```diff
@@ -1,343 +1,395 @@
+   in your :ref:`experiment configuration <experiment-configuration>`, such as:
+
+   -  If required, a custom Docker image is set in the experiment configuration.
+
+   -  ``pip install`` or ``apt install`` commands needed in the interactive environment are built
+      into a custom Docker image or included in the ``startup-hook.sh`` file in the model definition
+      directory root. See :ref:`startup-hooks` for more information.
+
+   -  Custom bind mounts required in the interactive environment are specified in the experiment
+      configuration.
+
+   -  Environment variables are correctly set in the experiment configuration.
+
+   If no customizations are missing, the following, new layers introduced with a cluster-managed
+   experiment could be the problem cause:
+
+   -  The ``checkpoint_storage`` settings are used for cluster-managed training. If
+      ``checkpoint_storage`` is not configured in the experiment configuration or the master
+      configuration, an error message can occur during experiment configuration validation before
+      the experiment or trials are created. Correct this by providing a ``checkpoint_storage``
+      configuration in one of the following locations:
+
+      -  :ref:`master-configuration`
+      -  :ref:`experiment-configuration`
+
+   -  For a cluster-based experiment, configured ``checkpoint_storage`` settings are validated
+      before training starts. The message ``Checkpoint storage validation failed``, indicates that
+      you should review the ``checkpoint_storage`` setting values.
+
+   -  The experiment configuration is more strictly validated for cluster-managed experiments than
+      for ``--local --test`` mode. Errors related to ``unmarshaling JSON`` when attempting to submit
+      the experiment to the cluster indicate that the experiment configuration has errors. Review
+      the :ref:`experiment configuration <experiment-configuration>`.
+
+If you are unable to identify the root cause of the issue, contact Determined `community support
 <https://join.slack.com/t/determined-community/shared_invite/zt-cnj7802v-KcVbaUrIzQOwmkmY7gP0Ew>`__!
 
-8. Does a single-GPU experiment work?
-=====================================
+.. _step8:
+
+****************************************************
+ Step 8 - Verify that a single-GPU experiment works
+****************************************************
+
+This step is similar to :ref:`Step 7 <step7>`, except that it introduces hyperparameter search and
+executes full training for each trial.
+
+#. Configure your system the same as :ref:`Step 7 <step7>`:
+
+   Confirm that your experiment configuration does not specify ``resources.slots_per_trial`` or that
+   it is set to ``1``. For example:
+
+   .. code:: yaml
+
+      resources:
+        slots_per_trial: 1
+
+#. Create an experiment without the ``--test`` or ``--local`` arguments:
+
+   You might find the ``--follow``, or ``-f``, arguments helpful:
```

Comment:
I do think `flag` is a more correct word than `argument`.  "Flags" generally don't have parameters (like `--verbose`), while "options" usually do (like `--output dump.txt`).  I think of "arguments" as either being positional arguments (neither flags nor options) or as a general term for tokens on the command line (where flags and options and positional arguments are all types of arguments).

In either case, "arguments" should not be plural because it's the same flag, just in a long form and a short form.

Reply:
I'm still leaning toward 'argument' (singular), as in Python's argparse(). 'Option' could also be ok for a CLI. If it was a program, it would be 'parameter'. I think 'flag' is the more general of all of those but usually refers to a programming variable, unofficially.

Let's have one go-around and pick one. They'll know what we mean.